typings/index.d.ts

```diff
@@ -2071,19 +2071,15 @@ export class ApplicationCommandManager<
     null
   >;
   private commandPath({ id, guildId }: { id?: Snowflake; guildId?: Snowflake }): unknown;
+  public create(command: ApplicationCommandData): Promise<ApplicationCommandType>;
   public create(command: ApplicationCommandData, guildId: Snowflake): Promise<ApplicationCommand>;
-  public create(command: ApplicationCommandData, guildId?: Snowflake): Promise<ApplicationCommandType>;
   public delete(command: ApplicationCommandResolvable, guildId?: Snowflake): Promise<ApplicationCommandType | null>;
+  public edit(command: ApplicationCommandResolvable, data: ApplicationCommandData): Promise<ApplicationCommandType>;
```

Comment:
Only to me this line seems too long 😅? Did you try to run prettier?

Reply:
It's the same length as the line above, which wasn't modified.